src/MutableVertexPartition.cpp

```diff
@@ -306,6 +306,48 @@ vector<size_t> MutableVertexPartition::renumber_communities(vector<MutableVertex
  Renumber the communities using the original fixed membership vector. Notice
  that this doesn't ensure any property of the community numbers.
 *****************************************************************************/
+void MutableVertexPartition::renumber_communities(map<size_t, size_t> const& membership)
+{
+  // Skip whole thing if there are no fixed nodes for efficiency
+  if (membership.size() == 0)
+    return;
+
+  // The number of communities does not depend on whether some are fixed
+  size_t nb_comms = n_communities();
+
+  // Fill the community map with the original communities
+  vector<size_t> comm_map(nb_comms);
```

Comment:
Perhaps a bit confusing to call this a `map`, given that the variable `membership` is actually a `map`. Perhaps simply label this something like `new_comm_id` similar to the other `renumber_communities` function.

Reply:
done